### DISCUSSION TITLE
[@types/mailparser] from

### DISCUSSION THREAD

**Discussion by gabrc52** [DefinitelyTyped/DefinitelyTyped#68304] [Issues with a @types package]
![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/24363938/ddfa6a0e-0614-4845-be46-af8bb88d163a)

In `EmailAddress` in `@types/mailparser`, did you really mean to make the address not optional but the name mandatory? There are emails with no name field, and I assume every email needs to have a sender? So this seems like a typo.

[a comment by an automated account is not shown]

**Reply by peterblazejewicz** [reply]
@gabrc52 yes, it looks the intent of the the type here is exactly as you said. But please compare with native module usage. Type declarations here are 'best-effort' to reflect underlying JS native code, done as community effort, and can sometime be incorrect (so I'd go and check with JS module details)
